cruise-control-core/src/main/java/com/linkedin/cruisecontrol/monitor/sampling/aggregator/Imputation.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2017 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.cruisecontrol.monitor.sampling.aggregator;
+
+/**
+ * There are a few imputations we will do when there is not sufficient samples in a window for a
+ * partition. The imputations are used in the following preference order.
```

Comment:
When does the CC consider the samples as insufficient and resort to imputations? -- e.g. when at least one sample is missing?